### DISCUSSION TITLE
Unable to write from Pandas Dataframe to Redshift Table

### DISCUSSION THREAD

**Discussion by adiyer27** [aws/amazon-redshift-python-driver#48] [Q&A] [answered]
Hello,

I am using the write_dataframe() method to write a table into a Redshift Cluster.
The examples in the documentation seem to be using temp tables and I am using normal ones.
While my code executes without any issue, I dont see the table on my cluster (Not accessing via Python but via my IDE). Anything I am missing?
My code is below:

with redshift_connector.connect(
  _connection details_
) as conn:
    with conn.cursor() as cursor:
        cursor.execute("drop table if exists schema.table_name")
        cursor.execute("create table  schema.table_name(column A varchar, Column B float4)")
        cursor.write_dataframe(my_dataframe, " schema.table_name")

**Comment by Brooke-white** [comment, marked as answer]
Hi @adiyer27 ,

Apologies for the late response -- the cause for this issue is that `conn.autocommit` is set to false by default, so you must manually commit using `conn.commit()` or enable autocommit by setting `conn.autocommit = True`.
